builds/jazz-build-module/aws-lambda-module.groovy

```diff
@@ -0,0 +1,48 @@
+/**
+ * Query AWS for the ARN of a lambda
+ * 
+ * @param stackName Stack name used to query for the lamba ARN.
+ * @return object with the ARN for the lambda.
+ */
+def getLambdaARN (stackName) {
+	def arn = "";
+
+	try {
+		def cloudformation_resources = "";
```

Comment:
Fix spacing issue.

Reply:
Replaced tabs with spaces for next commit.